### PR TITLE
fixes: Simulator Navbar disappears at the end of tutorials guide 

### LINKED ISSUE #2348: Bug - Simulator Navbar disappears at the end of tutorials guide
**Describe the bug**
The end msg of the tutorials guide goes out of the screen and makes the navbar disappear. reloading is required to restore everything in a normal state

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'circuitverse.org/simulator'
2. Click on 'help-->tutorials guide'
3. click next till the last msg appear

**Expected behavior**
msg must not go out of screen and navbar shall not disappear

**Screenshots**
image

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by vedant-jain03**
Fixes #2348 

#### Describe the changes you have made in this PR -

I have changed the styling of the issue button as it was overriding CSS of others, I convert its the position from absolute to fixed and also assigned it '!important' keyword on it's size

### Screenshots of the changes (If any) -
* Before 
Screenshot (58)
Screenshot (59)

* After
Screenshot (56)
Screenshot (57)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by vedant-jain03** [comment]
@pavanjoshi914 Please review my PR

**Comment by pavanjoshi914** [comment]
please correct  issues encountered by code climate

File: app/assets/stylesheets/simulator.scss [hunk 1]
```diff
@@ -22,16 +22,16 @@
 }
 
 .report-sidebar a {   
-  color: $white;
+  color: #fff;
```
Comment (pavanjoshi914) [inline app/assets/stylesheets/simulator.scss:25]: don't use hex  keep using variables instead

**Comment by vedant-jain03** [comment]
@pavanjoshi914 Please review

**Comment by pavanjoshi914** [comment]
LGTM 

@tachyons

**Comment by tachyons** [comment]
Please update PR with more meaningful title

**Comment by vedant-jain03** [comment]
@tachyons Kindly Review

**Pull request merged**